Sources/RealDeviceMap/Modell/Weather.swift

```diff
@@ -437,9 +439,31 @@ class Weather: JSONConvertibleObject, WebHookEvent {
             new.fogLevel != old.fogLevel ||
             new.sELevel != old.sELevel ||
             new.severity != old.severity ||
-            new.warnWeather != old.warnWeather ||
-            fabs(new.latitude - old.latitude) >= 0.000001 ||
-            fabs(new.longitude - old.longitude) >= 0.000001
+            new.warnWeather != old.warnWeather)
+
+        if hasChanged == true {
+            return true
+        }
+
+        let dLatitude = fabs(new.latitude - old.latitude)
+        let dLongitude = fabs(new.longitude - old.longitude)
+
+        hasChanged = (dLatitude >= 0.000001 || dLongitude >= 0.000001)
+
+        if hasChanged == true {
+            return true
+        }
+
+        let oldUpdated = old.updated ?? 0
+        let newUpdated = UInt32(Date().timeIntervalSince1970)
+
+        hasChanged = ((newUpdated - (newUpdated % 3600)) > (oldUpdated - (oldUpdated % 3600)))
```

Comment:
I am not sure i understand your logic here.
Would it be sufficient, if you just add `Date().timeIntervalSince1970 - (old.updated  ?? 0) > 3600` to the original or?

Reply:
Isn't that just within the last hour, not since the top of the hour?

I had originally added it to the existing or, but swift complained about it being too complicated or something. Sounded like nonsense to me, but it wasn't happy. Had to break it down to how I have it now.

Reply:
so you calculate the value for top of the hour for newUpdated and oldUpdated and you compare it... RDM will always calculate until line 460, also when weather is not changing during hour?! :)